web/src/main/java/org/springframework/security/web/firewall/StrictHttpFirewall.java

```diff
@@ -435,9 +436,9 @@ public void setAllowUrlEncodedLineSeparator(boolean allowUrlEncodedLineSeparator
 	 * @see Character#isISOControl(int)
 	 * @see Character#isDefined(int)
 	 */
-	public void setAllowedHeaderNames(Predicate<String> allowedHeaderNames) {
+	public static void setAllowedHeaderNames(Predicate<String> allowedHeaderNames) {
```

Comment:
Please keep the methods non-static. We only need the public constants to be static.

Reply:
Done.